tools/molecule.spec

```diff
@@ -0,0 +1,53 @@
+%global srcname molecule
+%global pypiname molecule
+%global pkgname python3-%{srcname}
+%global forgeurl https://github.com/ansible-community/%{srcname}
+
+Name:    %{pkgname}
+Version: 0.1.0
+Release: 1%{?dist}
+Summary: ...
+%forgemeta
+URL:       %{forgeurl}
+Source:    %{forgesource}
+License:   BSD
+BuildArch: noarch
+
+BuildRequires: python3-devel
+BuildRequires: python3-wheel
+BuildRequires: python3-tox-current-env
+
+%global _description %{expand:
+...
+}
+
+%description %{_description}
+
+%package -n python3-%{srcname}
+Summary: %summary
+
+%py_provides python3-%{srcname}
+
+%description -n python3-%{srcname} %{_description}
+
+%prep
+%autosetup -p1 -n %{srcname}-%{version}
+
+%generate_buildrequires
+%pyproject_buildrequires -t
+
+%build
+%{pyproject_wheel}
+
+%install
+%{pyproject_install}
+
+%pyproject_save_files %{pypiname}
+
+%check
+%{tox} -e lint,py
```

Comment:
Linting in Fedora might not be desired. Linting packages especially on Rawhide tend to get updated. If this was an official Fedora specfile, this would even be against the guidelines. https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/#_linters

Reply:
Also, if you desire to use specific toxenvs, you might need to pass the toxenvs to `%pyproject_buildrequires -e` to get all the relevant dependencies.